addons/addon-base-raas/packages/base-raas-workflow-steps/lib/steps/provision-account/provision-account.js

```diff
@@ -208,6 +208,10 @@ class ProvisionAccount extends StepBase {
           subnetId: cfnOutputs.VpcPublicSubnet1,
           vpcId: cfnOutputs.VPC,
           encryptionKeyArn: cfnOutputs.EncryptionKeyArn,
+          onboardStatusRoleArn: cfnOutputs.OnboardStatusRoleArn,
+          cfnStackName: stackInfo.StackName,
```

Comment:
Out of curiosity, why is `cfnStackName` and `cfnStackId` needed?

Reply:
Id is just displayed to the user, they're mainly saved for consistency with the BYOB codebase. cfnStackName is used in the DescribeStacks call